src/worker/tasks/mineTransactionWorker.ts

```diff
@@ -257,6 +264,66 @@ const _mineUserOp = async (
   };
 };
 
+const _notifyIfLowBalance = async (transaction: MinedTransaction) => {
+  const { isUserOp, chainId, from } = transaction;
+  if (isUserOp) {
+    // Skip for userOps since they may not use the wallet's gas balance.
```

Comment:
Doesn't infinigods use smart wallet transactions?

Reply:
UserOps are primarily gas sponsored via Engine.
This check is useful for users sending non userOp transactions from backend wallets, which they're using too.